integrations/database/postgresql/install.sh

```diff
@@ -33,7 +33,10 @@ while ! nc -vz localhost ${LOCAL_PORT} > /dev/null 2>&1 ; do
     sleep 0.5
 done
 echo .
-psql -h localhost -U postgres -p ${LOCAL_PORT} -f kubearchive.sql
+psql -h localhost -U postgres -p ${LOCAL_PORT} -f setup.sql
+
+export PGPASSWORD="Databas3Passw0rd"
```

Comment:
We probably need the # notsecret comment in here

```suggestion
export PGPASSWORD="Databas3Passw0rd"  # notsecret
```